src/pages/MyPage/MyPage.js

```diff
@@ -5,15 +5,28 @@ import UserInfo from '../../components/MyPage/UserInfo';
 
 const MyPage = () => {
   const [contentList, setContentList] = useState([]);
+
   useEffect(() => {
     fetch('/data/contentData.json')
       .then(res => res.json())
       .then(contentData => setContentList(contentData));
   }, []);
 
+  const [userList, setUserList] = useState({});
+
+  useEffect(() => {
+    fetch('http://10.58.2.42:8000/users/mypage', {
+      headers: {
+        Authorization: localStorage.getItem('Authorization'),
+      },
+    })
```

Comment:
```suggestion
  useEffect(() => {
    fetch(`${BASE_URL}users/mypage`, {
      headers: {
        Authorization: localStorage.getItem('Authorization'),
      },
    })
```
현재 config 파일이 merge 된 상황입니다.
ip 주소를 이렇게 기입할 경우에는 ip 주소가 바뀔 때마다 전부 일일히 수정해 줘야하는 번거로움이 있습니다.
이런 번거로움을 config 파일을 이용해 구현해 줄 수 있겠습니다.

* 이해가 되지 않는다면 형겸님이나 저를 찾아주세요